modules/static/googlechrome/lighthouse/initref

```diff
@@ -1 +1 @@
-e7a6d39fafc69cb83a20bda589ea413f4825cb5a
+v10.3.0
```

Comment:
This is the release after this comment: https://github.com/GoogleChrome/lighthouse/pull/15116#issuecomment-1569261877